code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/GithubOAuthImpl.java

```diff
@@ -74,50 +85,15 @@ public Account getAccount(ApiRequest request) {
             return null;
         }
 
-        Account projectAccount = authDao.getAccountById(new Long(unobfuscatedId));
-
-        if (validateProjectAccount(projectAccount, token)) {
-            return projectAccount;
+        project = authDao.getAccountById(new Long(unobfuscatedId));
```

Comment:
James, is there a chance that the project is null at this point? If yes, we should probably add a not null check for it.